.pre-commit-config.yaml

```diff
@@ -9,11 +9,13 @@ repos:
     rev: 3.9.2
     hooks:
       - id: flake8
+        additional_dependencies: [flake8-annotations]
```

Comment:
We shouldn't need additional exclusions here if the `.flake8` ones are good.

Reply:
I thought that too. But it looks like when running `flake8` via `pre-commit` the exclusions in `.flake8` are ignored :man_shrugging: Would be happy to see only one single source of truth.

Reply:
That was not my experience with the main repo. I suspect what's happening is partial vs. full paths being passed. Try fragments like e.g. `*/tests/fixtures/*/setup.py`. Some experimentation is likely needed.

Reply:
Ok, I've got it now. I removed the exclusion definitions completely from from yml file and make the `.flake8` to the singe source if truth.